React/Views/RCTViewManager.m

```diff
@@ -54,7 +54,7 @@ - (dispatch_queue_t)methodQueue
   return _bridge.uiManager.methodQueue;
 }
 
-- (UIView *)viewWithProps:(NSDictionary *)props
+- (UIView *)viewWithProps:(NSDictionary *) __attribute__((unused)) props 
```

Comment:
We've fixed this already internally, but for future reference you can just say:

  - (UIView *)viewWithProps:(__unused NSDictionary *)props